docs/rum_mobile_vitals.md

```diff
@@ -0,0 +1,41 @@
+| Application not responding     | When the UI thread of an application is blocked for more than 5 seconds, an `Application Not Responding` ([ANR][8]) error triggers. If the application is in the foreground, the system displays a dialog modal to the user, allowing them to force quit the application. <br /><br />   RUM tracks ANR occurrences and captures the entire stack trace that blocks the main thread when it encounters an ANR.                                                                                                                                                                                                                              |
+| Crash-free sessions by version | An [application crash][9] is reported due to an unexpected exit in the application typically caused by an unhandled exception or signal. Crash-free user sessions in your application directly correspond to  your end user’s experience and overall satisfaction. <br /><br />   RUM tracks complete crash reports and presents trends over time with [Error Tracking][10]. <br /><br />  With crash-free sessions, you can stay up to speed on industry benchmarks and ensure that your application is ranked highly on the Google Play Store.                                                                                                 |
+| CPU ticks per second           | High CPU usage impacts the [battery life][11] on your users’ devices and resource consumption.  <br /><br />  RUM tracks CPU ticks per second for each view and the CPU utilization over the course of a session. The recommended range is <40 for good and <60 for moderate.                                                                                                                                                                                                                                                                                                                                                         |
+| Memory utilization             | High memory usage can lead to [OutOfMemoryError][12], which causes the application to crash and creates a  poor user experience. <br /><br />  RUM tracks the amount of physical memory used by your application in bytes for each view, over the course of a session. The recommended range is <200MB for good and <400MB for moderate.                                                                                                                                                                                                                                                                                          |
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://developer.android.com/topic/performance/vitals
+[2]: https://raw.githubusercontent.com/DataDog/dd-sdk-android/alai97/android-monitoring-mobile-vitals/docs/images/mobile_vitals.png
```

Comment:
changed the image - can you review if it is okay @alai97 ?

Reply:
Can you send me the image on Slack @priyanshi-gupta? I don't see an updated image on here.